### DISCUSSION TITLE
[@types/google.maps] Property 'AutocompleteResponse' does not exist on type 'typeof places'.

### DISCUSSION THREAD

**Discussion by angelod1as** [DefinitelyTyped/DefinitelyTyped#65659] [Issues with a @types package] [answered]
I get the error `Property 'AutocompleteResponse' does not exist on type 'typeof places'.` but this property is present [in the documentation](https://developers.google.com/maps/documentation/javascript/reference/places-autocomplete-service#AutocompleteResponse).

The code which the error throws is:

```
export type GooglePlacePrediction =
  typeof google.maps.places.AutocompleteResponse.prototype.predictions[0]
```

[a comment by an automated account is not shown]

**Comment by angelod1as** [comment]
Updates?

**Reply by jeantimex** [reply, marked as answer]
The `AutocompleteResponse` type is [defined](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/google.maps/index.d.ts#L11403-L11408), however, it's just an interface, so `.prototype` won't work, because the interface has no implementation.

If you just want to get the type of the prediction, simply use `google.maps.places.AutocompletePrediction` instead.

**Reply by angelod1as** [reply]
That solves it, thanks :)
